Coucat/Pokedex
# Pokemon Card and Grid

# Pull Request

## Description

### Ticket impemented #[POK-19](https://coucat.atlassian.net/browse/POK-19)

[//]:<> (You can see the number of ticket in Jira in the top of task, replace the ticketNumber in the space and the number in the link)
### What did you implement it?
 
[//]:<> (Mark with an 'x' or a space)

- [ ] Bug Fix (Non-breaking change wich fixes an issue)
- [x] New Feature (Non-breaking change wich adds functionality)


### How did you implement it?
[//]:<> (In this section describe all things that you made)

The pokemon card was made follow the figma design with a little changes in some sizes of.
Add prop-types to use props in the pokemon card and set all things that we need like name, number or id, image and types or something that we need in the future.
Set a hover animation of up and down in the container.

## How can we test this ticket?
[//]:<> (Write in this part all the commands that you can use to test your code)
To see the hover animation you only need put the mouse over the container.
To test the props you can add a new pokemonCard in /components/pokemonCard/index.jsx and set the props as in the existing pokemonCard, also you can change the types in the same page in the object **types**. You can see all types in pokemonCard/styles.jsx. To see other image you only need change the number in the end of the link like:
 ```sh
https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/versions/generation-v/black-white/animated/{any-number}.gif
 ```
## Checklist
[//]:<> (Mark with a 'x' or a space )
